### PR TITLE
Remove rounded corners from dropdown separator

### PR DESCRIPTION
**Pull request by ettiennelouw**
Before
image

After
image

### CODE REVIEW

File: src/View/Components/Dropdown/DropdownItem.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 class DropdownItem extends Component
 {
-    public bool $separator;
+    public ?bool $separator;
```
Comment (ph7jack) [inline src/View/Components/Dropdown/DropdownItem.php]:
```suggestion
    public bool $separator;
```

File: src/View/Components/Dropdown/DropdownItem.php [hunk 2]
```diff
@@ -32,9 +32,9 @@ public function getClasses(): string
...
         return Str::of('text-secondary-600 px-4 py-2 text-sm flex items-center cursor-pointer rounded-md')
             ->append(' transition-colors duration-150 hover:text-secondary-900 hover:bg-secondary-100')
-            ->append(' dark:text-secondary-400 dark:hover:bg-secondary-700')
-            ->when($this->separator, function (Stringable $str) {
-                return $str->append(' border-t border-secondary-200 dark:border-secondary-600');
-            });
+            ->append(' dark:text-secondary-400 dark:hover:bg-secondary-700');
+            // ->when($this->separator, function (Stringable $str) {
+            //     return $str->append(' border-t border-secondary-200 dark:border-secondary-600');
+            // });
```
Comment (ph7jack) [inline src/View/Components/Dropdown/DropdownItem.php]: remove this block of code

File: resources/views/components/dropdown/item.blade.php [hunk 3]
```diff
@@ -1,3 +1,7 @@
+@if ($separator ?? '')
```
Comment (ph7jack) [inline resources/views/components/dropdown/item.blade.php]:
```suggestion
@if ($separator)
```

**Review by ph7jack — CHANGES_REQUESTED** [review]
Thanks for your contribution, I liked it
Could you fix these comments?

**Comment by ettiennelouw** [comment]
Awsome, oh wow yes sure! Forgot the commented code.

**Comment by ettiennelouw** [comment]
I have removed the commented code and code changes as requested.

**Review by ph7jack — APPROVED** [review]
Great

**Comment by ph7jack** [comment]
@ettiennelouw  I recommend you improve your commits security using the signed commits

https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Pull request merged**